src/metabase/api/query_description.clj

```diff
@@ -14,13 +14,21 @@
 
 (defn- get-aggregation-description
   [metadata query]
-  (let [field-name (fn [match] (:display_name (Field (mbql.u/field-clause->id-or-literal match))))]
```

Comment:
@paulrosenzweig this was the actual issue - `match` here was more than one match. it was `[[:field-id 10] [:field-id 14]]`

I just handled extra cases as part of the fix.